frontend/views/containers/chatroom/file-attachment/ChatFileAttachmentWarningModal.vue

```diff
@@ -32,7 +32,7 @@ export default {
         'large': {
           title: L('File too large'),
           // TODO: replace '1 GB' below with a value delivered from the server once implemented.
-          content: L('That file is too large and cannot be uploaded. The limit is {sizeLimit}', { sizeLimit: L('1 GB') })
+          content: L('That file is too large and cannot be uploaded. The limit is {sizeLimit}', { sizeLimit: L('6.29 MB') })
```

Comment:
A few issues here:

1. 6.29 MB is a constant that's specified right here. It should be retrieved from somewhere else.
2. If you're trying to translate "MB", then translate it like so: `L('{sizeInMegabytes} MB', { sizeInMegabytes })`
3. But don't do that. It makes no sense to have two L functions, one inside the other. Instead, do it like this: 

```js
L('That file is too large and cannot be uploaded. The limit is {sizeLimit} MB', { sizeLimit })
```

Reply:
> It should be retrived from somewhere else

I believe `CHAT_ATTACHMENT_SIZE_LIMIT` in `utils/constants.js` can be used for now for this.